sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -1,16 +1,17 @@
-Release History
+# Release History
```

Comment:
Do we want the `===================`s and `-------------------------`s?  I think this was original a `.txt` file and didn't get changed back from `.md` when we started this work before the last wave of releases.

Reply:
(Ditto for other Storage changelogs being switched from `.txt`)

Reply:
I removed all of those